superset-frontend/plugins/plugin-chart-echarts/src/Treemap/index.ts

```diff
@@ -17,7 +17,13 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { Behavior, ChartMetadata, ChartPlugin, t } from '@superset-ui/core';
+import {
+  Behavior,
+  ChartLabel,
+  ChartMetadata,
```

Comment:
```suggestion
  Behavior,
  ChartMetadata,
```
ChartLabel is not used in this plugin, so it's failing TSLint.

Reply:
Resolving TSLint issue